geocoder/admin0/data/wikipedia_countries_native_names.csv

```diff
@@ -46,15 +46,15 @@ TLS,East Timor,Timor Lorosa'e
 BHS,Bahamas,The Bahamas
 CHN,China,Zhōngguó
 EGY,Egypt,Misr or Masr
-,Equatorial Guinea,Guinea Ecuatorial
+GNQ,Equatorial Guinea,Guinea Ecuatorial

```

Comment:
This was not got from a raw source, just copy-pasting things from the Wikipedia. I only added official ISO3 codes, I avoided other ones that we're using but are not that official.